### PR TITLE
Fixing Gentoo CI QA problems

### PR DESCRIPTION
**Pull request by gentoo-setan**
Just improving stuff

---

Please check all the boxes that apply:

- [X] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [X] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [X] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [X] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

### CODE REVIEW

File: app-text/pdf2html/pdf2html-1.4.ebuild [hunk 1]
```diff
@@ -12,13 +12,14 @@ SRC_URI="ftp://atrey.karlin.mff.cuni.cz/pub/local/clock/pdf2html/${P}.tgz"
...
 
 DEPEND=">=media-libs/libpng-1.4
-	sys-libs/zlib"
+	sys-libs/zlib
+"
 RDEPEND="${DEPEND}
 	app-text/ghostscript-gpl
-	>=media-gfx/imagemagick-6"
+	>=media-gfx/imagemagick-6
+"
```
Comment (juippis) [inline app-text/pdf2html/pdf2html-1.4.ebuild:22]: These are usually maintainer-chosen styling preferences - you shouldn't touch this.

File: app-text/namazu/namazu-2.0.22_pre8.ebuild [hunk 2]
```diff
@@ -10,10 +10,10 @@ MY_P="${P/_pre/pre}"
 DESCRIPTION="Namazu is a full-text search engine"
 HOMEPAGE="http://www.namazu.org/"
 SRC_URI="http://www.namazu.org/test/${MY_P}.tar.gz"
+S="${WORKDIR}"/${MY_P}
 
 LICENSE="GPL-2+"
 SLOT="0"
-KEYWORDS=""
```
Comment (juippis) [inline app-text/namazu/namazu-2.0.22_pre8.ebuild:16]: Hmm, while this indeed prints a pkgcheck scan warning, removing this line makes using tools such as **ekeyword** impossible. I wonder what we should do here, I'll probably file a pkgcheck issue for it.

Comment (a17r) [reply app-text/namazu/namazu-2.0.22_pre8.ebuild:16]: NAK.

Comment (a17r) [reply app-text/namazu/namazu-2.0.22_pre8.ebuild]: It is a super minor notice not even rising to the level of a warning, just ignore it.

**Review by juippis — APPROVED** [review]

**Pull request closed without merging**